.gitignore

```diff
@@ -42,3 +42,5 @@ definitions/**/*.xlsx#
 
 # aat env file
 .aat-env
+
+cftlib_log_file_IS_UNDEFINED
```

Comment:
this needs re-removing

Reply:
@nilay913, @benouaer and I discussed this. Agreed that while `cftlib_log_file_IS_UNDEFINED` file is being produced when CCD test fails it would make sense to keep in .gitignore. 

See createPropertiesFolder method below for where file is created:
https://github.com/hmcts/rse-cft-lib/blob/main/cftlib/lib/bootstrapper/src/main/java/uk/gov/hmcts/rse/ccd/lib/LibRunner.java